Makefile

```diff
@@ -80,8 +80,25 @@ releases: release/access-slack release/access-jira release/access-mattermost rel
 .PHONY: build-all
 build-all: access-slack access-jira access-mattermost access-pagerduty access-gitlab access-email terraform event-handler
 
+.PHONY: update-api-version
+update-api-version:
+	@test $(VERSION)
+	# Update the teleport/api dependency to the given version.
+	# This will fail if teleport/api@VERSION has not been released.
+	# It can be manually updated to a specific git commit with
+	# go get github.com/gravitational/telepeport/api@v0.0.0-timestamp-6_byte_hex_commit_hash.
+	go get github.com/gravitational/telepeport/api@v$(VERSION)
+	go mod tidy
+
+	# Once the API version is updated, the terraform schema
+	# must be regenerated with the up to date .proto files.
+	# If significant changes have been made to the grpc API
+	# then the terraform/gen_teleport.yaml file will need to
+	# be updated manually.
+	$(MAKE) -C terraform gen-schema
```

Comment:
Does this only make sense in the context of `update-api-version` target, or would it make sense to have a separate target for it which `update-version` would depend on? Isn't this how you had it here before?

Reply:
I think it makes more sense in the context of `update-api-version`, since that's when the `.proto` file dependencies are updated. I don't think you'd ever want to do one without the other, which is why I moved it here.

Reply:
Would this be invoked like
```shell
$ make update-api-version VERSION=8.3.0
```
... or would we expect `VERSION` to be  set in an other makefile, or build environment?

Reply:
The former, because it will primarily be called when you run `make update-version VERSION=8.3.0`. That's how it's being done currently, though we could add `VERSION` to the makefile.